rskj-core/src/test/java/co/rsk/peg/PowpegMigrationTest.java

```diff
@@ -1403,6 +1396,54 @@ void test_change_powpeg_from_p2shErpFederation_with_mainnet_powpeg_post_RSKIP_35
         );
     }
 
+    @Test
+    void test_change_powpeg_from_p2shErpFederation_with_mainnet_powpeg_post_RSKIP_377_stores_standard_redeemScript() throws Exception {
```

Comment:
Seems like we are lacking an assertion that the standard p2sh redeem script was stored as expected.

Perhaps getting the value from the storage should be enough to validate it.

We should also check that pre-fingerrot, the non-standard p2sh script is stored. Perhaps this last assertion can be added as part of the other tests, so it's not necessary to create a new test just for this

Reply:
Done